src/test/java/com/rackspacecloud/metrics/ingestionservice/InfluxDBHelperTests.java

```diff
@@ -31,143 +35,170 @@
 @EmbeddedKafka(partitions = 1, topics = {IngestionServiceApplicationTests.UNIFIED_METRICS_TOPIC})
 public class InfluxDBHelperTests {
 
-    static final String UNIFIED_METRICS_TOPIC = "unified.metrics.json";
+    RestTemplate restTemplateMock;
+    RouteProvider routeProviderMock;
+    MeterRegistry meterRegistry;
+    InfluxDBUtils influxDBUtilsMock;
+    Timer influxDBWriteTimer;
+    Timer getInfluxDBInfoTimer;
 
     @Before
     public void setUp() {
+        restTemplateMock = mock(RestTemplate.class);
+        routeProviderMock = mock(RouteProvider.class);
+        meterRegistry = mock(MeterRegistry.class);
+        influxDBUtilsMock = mock(InfluxDBUtils.class);
+        influxDBWriteTimer = mock(Timer.class);
+        getInfluxDBInfoTimer = mock(Timer.class);
+        when(meterRegistry.timer("ingestion.influxdb.write")).thenReturn(influxDBWriteTimer);
+        when(meterRegistry.timer("ingestion.routing.info.get")).thenReturn(getInfluxDBInfoTimer);
     }
 
     @Test
     public void ingestToInfluxDb_withExistingDatabaseAndRetPolicy_shouldSucceed() throws Exception {
-        RestTemplate restTemplateMock = mock(RestTemplate.class);
-        RouteProvider routeProviderMock = mock(RouteProvider.class);
-        InfluxDBHelper influxDBHelper = new InfluxDBHelper(restTemplateMock, routeProviderMock);
+        InfluxDBHelper influxDBHelper = new InfluxDBHelper(
+                restTemplateMock, routeProviderMock, meterRegistry,
+                influxDBUtilsMock, 100, 100);
         String tenantId = "hybrid:1667601";
+        String measurement = "cpu";
         String databaseName = "existing_db";
         String rpName = "existing_rp";
 
         doReturn(getTenantRoutes(tenantId, databaseName, rpName))
-                .when(routeProviderMock).getRoute(tenantId, restTemplateMock);
+                .when(routeProviderMock).getRoute(tenantId, measurement, restTemplateMock);
 
-        System.out.println(String.format("Testing with tenantId:[%s]; databaseName:[%s]; rpName:[%s]",
+        System.out.println(String.format("Testing with tenantIdAndMeasurement:[%s]; databaseName:[%s]; rpName:[%s]",
                 tenantId, databaseName, rpName));
 
-        successfulIngestionTest(influxDBHelper, restTemplateMock, tenantId, databaseName, rpName);
+        successfulIngestionTest(influxDBHelper, restTemplateMock,
```

Comment:
I don't see any assertions or Mockito verify calls. Is there some other way the unit tests are verifying success?

Reply:
Added Mockito.verify()